youtube_dl/extractor/pbs.py

```diff
@@ -466,7 +467,7 @@ def _extract_webpage(self, url):
                 url, display_id, note='Downloading player page',
                 errnote='Could not download player page')
             video_id = self._search_regex(
-                r'<div\s+id="video_([0-9]+)"', player_page, 'video ID')
+                [r'<div\s+id="video_([0-9]+)"', r'"id":[\s]*"([0-9]+)"'], player_page, 'video ID')
```

Comment:
`[]` is pointless. Must be more relaxed.

Reply:
What ? I need to use [] for making a list like you want.

Reply:
I'm not about list.

Reply:
So execuse me but I don't understand your comment.
What do I have to change?

Reply:
Sorry I thought you were talking about the `[]` around the 2 regex.